object/conf.go

```diff
@@ -45,6 +45,8 @@ var (
 	UseOAuthProxy              = false
 	DefaultUploadFileQuota     = 50
 	Domain                     = "forum.casbin.com" // domain
+	AutoSyncFromGoogleGroup    = true
+	AutoSyncPeriodSecond       = 30
```

Comment:
AutoSyncFromGoogleGroup can be removed. Use `AutoSyncPeriodSecond == -1` to represent `AutoSyncFromGoogleGroup == false`